draft-irtf-cfrg-opaque.md

```diff
@@ -1908,14 +1908,17 @@ implementation considerations.
   of the derivation of OPRF keys via a single PRF. As long as the derivation
   of different OPRF keys from a single OPRF has different PRF inputs, the
   protocol is secure. The choice of such inputs is up to the application.
-- {{JKX18}} comments on a strong defense against essentially the only
-  inherent remaining weakness of a "strong aPAKE", namely, offline
+- {{JKX18}} comments on a defense against offline
   dictionary attacks upon server compromise or honest-but-curious servers.
   The authors suggest implementing the OPRF phase as a Threshold OPRF {{TOPPSS}},
   effectively forcing an attacker to act online or to control at least t key
-  shares, where t is the threshold number of shares necessary to recombine
-  the secret OPRF key and run an offline dictionary attack. This implementation only affects the server and changes
-  nothing for the client. However, this mechanism is out of scope for this document.
+  shares (among the total n), where t is the threshold number of shares necessary
+  to recombine the secret OPRF key, and only then be able to run an offline dictionary
+  attack. This implementation only affects the server and change nothing for the client.
+  Furthermore, if the Threshold OPRF servers holding these keys are separate from
```

Comment:
```suggestion
  Furthermore, if the threshold OPRF servers holding these keys are separate from
```

Reply:
hm, I'm referring to the "Threshold OPRF {{TOPPSS}}" some lines above, where I used an uppercase T. Should I add the citation on this line or lowercase the top one?

Reply:
I think we should use lowercase threshold in both cases. It's just an adjective for the OPRF, not a proper noun.

Reply:
I committed the change, but then re-checked [in the Paper](https://eprint.iacr.org/2017/363.pdf), where they say "which we define as Threshold OPRF (T-OPRF)". I believe we should use uppercase to stick to the language used by the paper we're referring to

Reply:
I don't really agree. Again, there is not just one Threshold OPRF, there can be many, so I'm going to fix this up on main.

Reply:
the paper calls it 2HashTDH, later papers call it tdh-op.

Reply:
True, you're certainly right